### PR TITLE
Nrt monitoring cds quick

### PR DESCRIPTION
**Pull request by zackarno**
ok moved the relevant part of the analysis to a notebook for easier review...

So heres the deal:

- I was was working on `nrt-monitoring` branch for the email generation that used MARS data (for thresholds) and AWS data for NRT monitoring.
- Therefore, I branched off that to create this `nrt_monitoring-cds_quick` just in case the AWS data came online.
open

Assuming `process-cds-ecmwf` thresholds are okay then we can merge this into the `nrt-monitoring` and just replace all AWS/MARS data w/ CDs. Then we can have the large pipeline `nrt-monitoring` code reviewed as it is PR'd to main

The code to be reviewed is `analysis/04_April24_activation_status_copernicus.qmd` . I call some functions that I was using from the pipeline and have not refactored or moved them around, so let me know if anything needs clarification

### CODE REVIEW

File: analysis/04_April24_activation_status_copernicus.qmd [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+library(glue)
+library(gghdx)
+library(blastula)
+library(here)
+library(targets) # should remove for GHA eventually
+library(terra)
+gghdx()
+
+fp_email_util_funcs <- list.files(
+  file.path("src","email","email_utils"),
```
Comment (t-downing) [inline analysis/04_April24_activation_status_copernicus.qmd]: I just had to change this to `file.path("..", "src", "email", "email_utils")` but I presume you were going to refactor stuff anyways

**Review by t-downing — CHANGES_REQUESTED** [review]
As you suggested I only looked at `analysis/04_April24_activation_status_copernicus.qmd` and just had to change that one line to run it. As far as I can see other than that it's all good!

I had a quick look at the threshold calc as well and at a glance it looks good but I know you have another PR for that.

**Comment by caldwellst** [comment]
Yeah looks good to me.

File: analysis/04_April24_activation_status_copernicus.qmd (hunk 1 above)
Comment (zackarno) [reply analysis/04_April24_activation_status_copernicus.qmd]: good calll - addressed w/ `{here}` package: [48bf0ee](https://github.com/OCHA-DAP/ds-aa-lac-dry-corridor/pull/16/commits/48bf0ee6d8e5f5a763a7fd82b7ff0e2bc9f670ca)

**Pull request merged**